src/functions/function_tester.ts

```diff
@@ -0,0 +1,39 @@
+import { Env } from "../types.ts";
+import { FunctionContext, FunctionInvocationBody } from "./types.ts";
+
+type SlackFunctionTesterArgs<InputParameters> = {
+  inputs: InputParameters;
+  env?: Env;
+  token?: string;
+  event?: FunctionInvocationBody["event"];
+};
```

Comment:
nit: I'm thinking that we should tie this a bit closer to the `FunctionContext` itself and we could leverage a `Partial` to make things optional. I'm thinking something like

```suggestion
import type { FunctionContext } from "./types.ts";

type SlackFunctionTesterArgs<InputParameters> =
  & Partial<FunctionContext<InputParameters>>
  & {
    inputs: InputParameters;
  };
```

Reply:
Good call, helps it stay in sync w/ that type.